sdk/src/main/scala/com/horizen/storage/ForgingBoxesMerklePathStorage.scala

```diff
@@ -0,0 +1,85 @@
+package com.horizen.storage
+
+import com.horizen.SidechainTypes
+import com.horizen.consensus.ConsensusEpochNumber
+import com.horizen.utils.{BoxMerklePathInfo, BoxMerklePathInfoSerializer, ByteArrayWrapper, ListSerializer, Pair}
+import scorex.util.ScorexLogging
+import scorex.crypto.hash.Blake2b256
+
+import scala.collection.JavaConverters._
+import scala.util.{Failure, Random, Success, Try}
+import java.util.{ArrayList => JArrayList}
+
+
+class ForgingBoxesMerklePathStorage(storage: Storage) extends SidechainTypes with ScorexLogging
```

Comment:
comment with version update logic description is required.

Reply:
agree